kvmagent/kvmagent/test/test_migrate_domain_xml.py

```
@@ -0,0 +1,61 @@
+        <host name='172.24.1.1' port='6789'/>
+      </source>
+      <target dev='hdc' bus='ide'/>
+      <readonly/>
+    </disk>
+  </devices>
+</domain>
+"""
+        disks, dest_xml = self._build_domain_new_xml(domain_xml)
+
+        self.assertEqual([], disks)
+        root = ET.fromstring(dest_xml)
```

Ruff (0.15.15)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)